MaaSSim/utils.py

```diff
@@ -115,14 +115,14 @@ def load_G(_inData, _params=None, stats=True, set_t=True):
 def download_G(inData, _params, make_skims=True):
     # uses osmnx to download the graph
     print('Downloading network for {} witn osmnx'.format(_params.city))
-    inData.G = ox.graph_from_place(_params.city, network_type='drive')
+    full_graph = ox.graph_from_place(_params.city, network_type='drive')
```

Comment:
good job, I like it. 

See if tests with downloading networks (test_masssim_utils.py/test_networkIO is OK with it)

Reply:
you may trace the statistics and see how many nodes we have left:
`log.info('before {}nodes after {}nodes'.format(len(full_graph.nodes),len(inData.G.nodes))`

Reply:
> you may trace the statistics and see how many nodes we have left:
> `logger.info('before {}nodes after {}nodes'.format(len(full_graph.nodes),len(inData.G.nodes))`

To be included in utils.py so that it shows for every simulation?

Reply:
> good job, I like it.
> 
> See if tests with downloading networks (test_masssim_utils.py/test_networkIO is OK with it)

This test is run when I draft a pull request right?